### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by alidah0**
relates #195

- added new technologies 
- added gifs to MVP section
- made some edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
 
 ## The MVP :rainbow:
 
- GIF Image
+ - Web
+ 
+![](https://i.imgur.com/KtYlkTi.gif)
+
+![](https://i.imgur.com/CaNT7yA.gif)
```
Comment (mFlaifel) [inline README.md:111]: good picture, but I have a silly suggestion to try to crop the black line at the top.

File: README.md [hunk 2]
```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
...
+
+
+
+- Mobile
+![](https://i.imgur.com/0MXdmo2.gif)
+
```
Comment (mFlaifel) [inline README.md:117]: If you don't want to insert a mobile UX gif, remove this line.

File: README.md [hunk 3]
```diff
@@ -127,14 +150,12 @@ we will use `Heroku` Platform to deploy our website.
 - MongoDB
 - mongoose
 
+## Stretch Goals :goal_net:
 
-## Team :busts_in_silhouette:
-
-### Team Lead
-  - Asmaa Thabet  
+- Send a notifications from admin to applicants.
```
Comment (mFlaifel) [inline README.md:155]:
```suggestion
- Send notifications from admin to applicants.
```

File: README.md [hunk 4]
```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
...
+ - Web
+ 
+![](https://i.imgur.com/KtYlkTi.gif)
+
+![](https://i.imgur.com/CaNT7yA.gif)
+
+
+
+- Mobile
+![](https://i.imgur.com/0MXdmo2.gif)
```
Comment (Asmaathabet) [inline README.md:116]: There is No gif photo for mobile  [fix it]

File: README.md (hunk 1 above)
Comment (Asmaathabet) [inline README.md:111]: In web it should be two GIFS one for the dashboard and the other for the applicant, I just see one here

File: README.md [hunk 5]
```diff
@@ -92,7 +104,17 @@ The Admin sign in through sign-in page and he will be redirected to the dashboar
...
+
  
 ## Application Setup :mag:
```
Comment (Asmaathabet) [inline README.md:119]: In Application Setup you should write more about Mongo configurations, what should the person do to run database, and where to go. 
Just simple and short steps: 

starting with [create account ](https://account.mongodb.com/account/login?n=%2Fv2%2F5ddcde7c79358e92652b7c3a&nextHash=%23metrics%2FreplicaSet%2F5ddce29bbc1dc0c78af6810e%2Fexplorer%2Ftest%2Fbooks%2Ffind) for example

File: README.md (hunk 1 above)
Comment (Asmaathabet) [reply README.md:111]: good Notice!

**Comment by tomduggan85** [comment]
Hey @alidah0 and team - I have one piece of advice that probably doesn't matter as much for this project but is helpful in real-world projects which have lots of developers joining projects after they've been created and have matured: I always try to put the "application setup" at the top of the readme, because that's usually the most important piece of information for someone who is new to the repo.

**Pull request merged**